data_pipelines/actions/aws.py

```diff
@@ -0,0 +1,47 @@
+import os
+# from dotenv import load_dotenv
+from airflow.exceptions import AirflowSkipException
+from data_pipelines.connections.core import CSVConn
+from data_pipelines.connections.aws import S3Conn
+from data_pipelines.actions.core import SourceToSink
+
+class CSVToS3(SourceToSink):
+    source_class = CSVConn
+    sink_class = S3Conn
+
+    def run(self):
+
+        with self.source:
+            data = self.get_data()
+            data = [*data]
+            if data:
+                with self.sink:
+                    self.load_data(data)
+                self.log.info('Data Successfully Loaded!')
+            else:
+                raise AirflowSkipException('No Data Found on that date !')
+
+
+# if __name__ == '__main__':
```

Comment:
I think the code would be a lot cleaner if we got rid of these development test. 
What do you think?